2005-11-21  Jakub Jelinek  <jakub@redhat.com>

	* jni/classpath/jnilink.c (LINK_ReallyLinkClass,
	LINK_ReallyLinkKnownClass): Add dummy functions.

--- libjava/jni/classpath/jnilink.c.jj	2005-11-21 15:36:51.000000000 +0100
+++ libjava/jni/classpath/jnilink.c	2005-11-21 18:12:35.000000000 +0100
@@ -45,6 +45,26 @@ exception statement from your version. *
 #define GETCLASS(c) *(jclass*)(c)
 
 JNIEXPORT jclass JNICALL
+LINK_ReallyLinkClass	(JNIEnv * env, linkedClass * c, char * name)
+{
+	(void) env;
+	(void) c;
+	(void) name;
+	abort ();
+	return NULL;
+}
+
+JNIEXPORT jclass JNICALL
+LINK_ReallyLinkKnownClass (JNIEnv * env, linkedClass * c, jclass newClass)
+{
+	(void) env;
+	(void) c;
+	(void) newClass;
+	abort ();
+	return NULL;
+}
+
+JNIEXPORT jclass JNICALL
 LINK_RelinkClass     (JNIEnv * env, linkedClass * c, char * name) {
 	jclass found;
 	LINK_UnlinkClass(env,*c);
